runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttClientFactory.java

```diff
@@ -274,6 +275,7 @@ public final class MqttClientFactory implements MqttStreamFactory
     private final MqttPublishHeader mqttPublishHeaderRO = new MqttPublishHeader();
 
     private final MqttConnectV5FW.Builder mqttConnectV5RW = new MqttConnectV5FW.Builder();
+    private final MqttConnectHeaderFW.Builder mqttConnectHeaderRW = new MqttConnectHeaderFW.Builder();
```

Comment:
1. Use `MutableInteger` instead of `AtomicInteger` below and `.value = ...` instead of `.set(...)`.

Reply:
I'm not sure if you wanted to add this comment here.